backend/oasst_backend/user_repository.py

```diff
@@ -161,10 +161,10 @@ def query_users(
         users = users.order_by(User.display_name)
 
         if gt:
-            users = users.filter(User.display_name > gt)
+            users = users.filter(User.id > gt)
```

Comment:
Not sure what the intention here is, but it doesn't look right.

Reply:
I thought we want to cursor based on the user id?

Reply:
Is the user id column monotonically increasing?

Reply:
UUID V4 isn't guaranteed to be monotonically increasing.  I think only UUID V6 or V7 have that property.  V4 is random in it's ordering.